packages/core/components/Legend/Legend.Gradient.tsx

```diff
@@ -38,7 +38,7 @@ const LegendGradient = ({
 
   const numTicks = colors?.length
 
-  const longestLabel = labels && labels.length > 0 ? labels.reduce((a, b) => (a.length > b.length ? a : b)) : ''
+  const longestLabel = (labels || []).reduce((a: string, b) => (a.length > String(b).length ? a : b), '')
```

Comment:
@Atash3000 @adamdoe  I made this change to fix an issue one of the wcms devs was seeing. labels was coming back from the api with a null value in the array and null.length isn't a function. stringifying it makes sure it will at least show instead of breaking COVE